fil_fungible_token/src/token/state.rs

```diff
@@ -62,17 +63,14 @@ impl TokenState {
         // Load the actor state from the state tree.
         match bs.get_cbor::<Self>(cid) {
             Ok(Some(state)) => Ok(state),
-            Ok(None) => Err(anyhow!("No state at this cid {:?}", cid)),
+            Ok(None) => Err(anyhow!("no state at this cid {:?}", cid)),
             Err(err) => Err(anyhow!("failed to get state: {}", err)),
         }
     }
 
     /// Saves the current state to the blockstore, returning the cid
-    /// TODO: Potentially should replaced with more targeted saving of different branches of the state tree for
-    /// efficiency
-    /// - possibly keep track of "dirty" branches that need to be flushed and put into the blockstore
-    /// on save
-    /// - hack for now is that each method that touches a hamt, will itself commit the changes to the blockstore
+    /// TODO: should replaced with more targeted saving of different branches of the state tree for efficiency
+    /// i.e. only save the balances HAMT if it has changed, only save the allowance HAMTs if they have changed
```

Comment:
I *think* this is going to do the right thing anyway. But maybe write an issue rather than TODO here